pxr/usdImaging/lib/usdviewq/appController.py

```diff
@@ -1222,6 +1222,15 @@ def _rendererPluginChanged(self, plugin):
                     if action.text() == self._stageView.rendererPluginName:
                         action.setChecked(True)
                         break
+                # Then display an error message to let the user know something
+                # went wrong, and disable the menu item so it can't be selected
+                # again.
+                for action in self._ui.rendererPluginActionGroup.actions():
+                    if action.pluginType == plugin:
+                        self.statusMessage(
+                            'Renderer not supported: %s' % action.text())
+                        action.setText(action.text() + " (unsupported)")
+                        action.setDisabled(True)
```

Comment:
Not a huge deal, but I think we can 'break' here, like above?

Reply:
Yes, certainly.